docker/README.md

```diff
@@ -46,47 +50,44 @@ docker run -d \
   minimega
 ```
 
-The container runs the `start-minimega.sh` script as PID 1, which takes care of
-starting openvswitch, miniweb, and finally minimega. This means the minimega
-logs will be available in the container logs via Docker.
-
----
+The container runs the `start-minimega.sh` script as PID 1, which takes care of starting openvswitch, miniweb, and finally minimega. This means the minimega logs will be available in the container logs via Docker (`docker logs minimega`).
 
-# Using docker-compose
 
-### Install docker-compose
+# Using Docker Compose
 
-```bash
-VERSION=`git ls-remote https://github.com/docker/compose | grep refs/tags | grep -oP "[0-9]+\.[0-9][0-9]+\.[0-9]+$" | sort | tail -n 1`
-sudo curl -ksL "https://github.com/docker/compose/releases/download/${VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-sudo chmod +x /usr/local/bin/docker-compose
-```
+If you followed the [Docker installation instructions](https://docs.docker.com/engine/install/), then `docker compose` should already be installed. Verify this by running `docker compose version`.  If it's not, then install it: `sudo apt install docker-compose-plugin`
 
-### Start the minimega docker container
+Start the minimega Docker container with Docker Compose:
 
 ```bash
-docker-compose up -d
+cd docker/
+docker compose up -d
+docker compose logs -f  # CTRL+C to stop following logs
 ```
 
----
 
 # Extras
 
-### Convenience aliases
+## Convenience aliases
 
 ```bash
 cat <<EOF >> ~/.bash_aliases
-alias minimega='docker exec -it minimega minimega '
+alias mm='docker exec -it minimega minimega -e'
+alias mminfo='mm .columns name,state,ip,snapshot,cc_active vm info'
```

Comment:
Maybe want `vm info summary` here, but this is fine too

Reply:
`vm info summary` doesn't show IPs, but i'll add a new alias

Reply:
That's fine. Just interesting to see your preferred columns vs what's in `vm info summary`. Made me wonder if summary should be updated.

Reply:
@jacdavi I added `mminfo` as an alias to our internal scripts many years ago before `vm info summary` existed, and those were the fields we were most interested in. We just never switched over to summary and mminfo stuck around this whole time :) IMHO `id,name,state,ip,vlan,cc_active` would be a great summary nowadays, but it's pretty subjective and pry not worth code changes since it's easy to add aliases.